support/kind/kind.go

```diff
@@ -153,12 +153,12 @@ func (k *Cluster) Destroy() error {
 
 	p := k.e.RunProc(fmt.Sprintf(`kind delete cluster --name %s`, k.name))
 	if p.Err() != nil {
-		return fmt.Errorf("kind: delete cluster failed: %s: %s", p.Err(), p.Result())
+		return fmt.Errorf("kind: delete cluster %v failed: %s: %s", k.name, p.Err(), p.Result())
 	}
 
 	log.V(4).Info("Removing kubeconfig file ", k.kubecfgFile)
 	if err := os.RemoveAll(k.kubecfgFile); err != nil {
-		return fmt.Errorf("kind: remove kubefconfig failed: %w", err)
+		return fmt.Errorf("kind: remove kubefconfig %v failed: %w", k.kubecfgFile, err)
```

Comment:
Suggestion: same thing here, use exact type for template argument and the rest of the changes.